index.html

```diff
@@ -28,344 +28,245 @@
       <nav>
         <a href="./index.html" id="logo">Idrissa</a>
         <div class="icons">
-          <img src="./img/menu.svg" class="display-img" alt="" />
-          <img src="./img/Icon.png" class="no-display-img" alt="" />
-        </div>
-        <div class="ul-list">
-          <ul>
-            <li><a href="#portfolio" class="ul-link">Portfolio</a></li>
-            <li><a href="#about" class="ul-link">About</a></li>
-            <li><a href="#contact" class="ul-link">Contact</a></li>
-          </ul>
+          <img src="./img/menu.svg" class="display-open-button" alt="" />
+          <img src="./img/Icon.png" class="display-close-button" alt="" />
         </div>
+        <ul class="lists">
+          <li><a href="#portfolio" class="ul-link">Portfolio</a></li>
+          <li><a href="#about" class="ul-link">About</a></li>
+          <li><a href="#contact" class="ul-link">Contact</a></li>
+        </ul>
       </nav>
     </header>
 
-    <main class="content">
-      <section class="subcontent">
-        <h1>
-          I'm Idrissa Omar <br />
-          Glad to see <br />
-          you!
-        </h1>
-
+    <section class="content">
+      <div class="sub-content">
+        <h1>I'm Idrissa Omar</h1>
+        <h2>Glad to see you!</h2>
         <p>
-          I'm a software developer! I can help you build a product , feature or
+          I’m a software developer! I can help you build a product , feature or
           website Look through some of my work and experience! If you like what
-          you see and have a project you need coded, don't hestiate to contact
+          you see and have a project you need coded, don’t hestiate to contact
           me.
         </p>
+      </div>
+      <div class="social-media">
+        <p>let's connect</p>
+        <ul>
+          <li>
+            <a href="#">
+              <img src="img/linkedin.png" alt="linkedin logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/github.png" alt="github logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/medium.png" alt="medium logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/twitter.png" alt="twitter logo" />
+            </a>
+          </li>
+          <li>
+            <a href="#">
+              <img src="img/angellist.png" alt="angellist logo" />
+            </a>
+          </li>
+        </ul>
+      </div>
+    </section>
+    <section class="popup" data-popup-id="myPopup">
+      <div class="popup-content">
+        <div class="title popup-title">
+          <h2 class="popup-h2">Tonic</h2>
+          <i class="fas fa-times close-popup" data-dismiss-popup></i>
+          <div class="canopy popup-canopy">
+            <h4>Canopy</h4>
+            <ul>

```

Comment:
- [ ] Please make your popup close to the Figma design, some don't match the Figma design yet: 
- Kindly blur the sub background when the popup is open, it will make the popup more like Figma 👍🏿
-  It would be better if the background is not scrollable when the popup page is open. you can use CSS property `overflow: hidden;` to prevent this.



- [ ]  Please add more padding-top between the list info and the popup image as provided in Figma:


- Your Implementation:
 
![menu322](https://user-images.githubusercontent.com/79058364/170542559-2a16966f-3f68-43ab-a548-374c53636f14.png)

 - Figma Design:

![today3](https://user-images.githubusercontent.com/79058364/163398933-541800fb-decb-4be8-ba54-2b19f247f3c0.png)


-  Adding these features will make your design look great and look exactly like the Figma design. 💯 😘

Reply:
- [ ] As requested by the previous reviewer here 👇🏻 , Kindly make sure to address the requested changes :wink: 💯 and remember your ` <section class="popup" data-popup-id="myPopup">` section must stay empty you must remove everything inside I hope this is more clear now👍🏻 . If you have any questions please do not hesitate to reach out on slack or tag me on @BenMukebo 😄

Reply:
- [OPTIONAL] Great job so far 👏🏻 😘  your design looks great and like the Figma design. but you can still improve it by:
- Blurring the sub background when the popup is open, it will make the popup more like Figma 👍🏿 and
- Preventing the background not to being scrollable when the popup page is open. you can use CSS property overflow: hidden; to prevent this.